docs/local-development.md

```diff
@@ -3,7 +3,7 @@
 ## Pre-reqs
 
 * Access to a 4.3+ Openshift cluster
-* A clone of [korekuta-go-operator](https://github.com/project-koku/koku-metrics-operator)
+* A clone of [korekuta-go-operator](https://github.com/project-costmanagement/costmanagement-metrics-operator)
```

Comment:
We should update `korekuta-go-operator`. Didn't realize we never changed this reference.

Reply:
But on a different note, should we just exclude the docs folder in this branch? I'd be ok with adding this to the `remove files` in the Makefile.

Reply:
Added to the remove files